Source/WebKit/WPEPlatform/wpe/wayland/WPEWaylandSHMPool.h

```diff
@@ -39,11 +39,12 @@ class WaylandSHMPool {
     WaylandSHMPool(void*, size_t, WTF::UnixFileDescriptor&&, struct wl_shm*);
     ~WaylandSHMPool();
 
-    void* data() const { return m_data; }
-    size_t size() const { return m_size; }
+    std::span<uint8_t> mutableSpan() LIFETIME_BOUND { return unsafeMakeSpan<uint8_t>(static_cast<uint8_t*>(m_data), m_size); }
```

Comment:
This could be `private`, because it's used only internally in the implementation of the `write()` method.